api/controllers/game/rematch.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Indicate that the player wants to play again
+ */
+const gameAPI = sails.hooks['customgamehook'];
+module.exports = async function (req, res) {
+  try {
+    const { usr: userId, pNum } = req.session;
+    const { gameId, rematch } = req.body;
+
+    try {
+      await Game.findOne({ id: gameId }).populate('players');
+    } catch (err) {
+      if (rematch === false) {
+        return res.ok();
+      }
+    }
```

Comment:
Is this stale? It looks like the query is not used and then we immediately requery for the game anyway
```suggestion
```